main/options/options.h

```diff
@@ -93,7 +93,7 @@ enum Phase {
     INIT,
     PARSER,
     DESUGARER,
-    DSL,
+    SyntaxTransform,
```

Comment:
This also should probably be `SYNTAX_TRANSFORM` to match style with the rest of this enum.